Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2843,19 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
+     * the admin screen.
+     */
+    public final function setIsShowMosaicButton($isShown)
```

Comment:
Please change the method name to `showMosaicButton` : showing is already a boolean thing, because you either show or do not show. It's not really a setter because you are also unsettings things with it.